src/SprykerEco/Yves/Heidelpay/Theme/default/views/direct-debit/direct-debit.twig

```diff
@@ -0,0 +1,39 @@
+{% extends model('template') %}
+
+{% define data = {
+    form: form,
+} %}
+
+{% block body %}
+    <div class="payment-method spacing-left">
+        {{ 'payment.heidelpay.pay_with_direct_debit' | trans }}
+        {{ form_errors(data.form.heidelpayDirectDebit) }}
+        <div class="grid spacing-top">
+            <ul class="col col--xl-9 col--lg-12">
+                {% for paymentOption in data.form.heidelpayDirectDebit.selected_payment_option %}
+                    {% set isPaymentOptionActive = (paymentOption.vars.value == 'direct-debit-existing-registration') ? true : false %}
```

Comment:
Please don't use ternary with a boolean condition, because `paymentOption.vars.value == 'direct-debit-existing-registration'` already returns you `true` or `false`.

Reply:
fixed